pages/common/pngquant.md

```diff
@@ -0,0 +1,36 @@
+# pngquant
+
+> PNG converter and lossy image compressor.
+> More information: <https://pngquant.org/>.
+
+- Compress a PNG as much as possible and write result to a new file:
```

Comment:
There are multiple fixes available: please chose the most appropriate one.

```suggestion
- Compress a specific PNG as much as possible and write result to a new file:
```

Optional change: `a specific` may sound better.

```suggestion
- Compress a specific .png as much as possible and write result to a new file:
```

Optional change: you can use `.ext` extension format instead of `EXT` one.